Makefile

```diff
@@ -119,7 +119,7 @@ chrome: wget
 		ARCHIVE=google-chrome-stable_current_amd64.deb; \
 		wget -q https://dl.google.com/linux/direct/$${ARCHIVE}; \
 		sudo dpkg --install $${ARCHIVE} 2>/dev/null || true; \
```

Comment:
```suggestion
		sudo apt-get update; \
		sudo dpkg --install $${ARCHIVE} 2>/dev/null || true; \
```

Reply:
I *think* that the index of package locations is stale. That is why it fails with the 404.

I tried adding `sudo apt-get update` and it seems to work.
https://github.com/web-platform-tests/wpt.fyi/pull/2992

Reply:
I thought about running 'apt update', but I assumed it wouldn't help given that we essentially initialize a new docker container every time. Your run did succeed however, so maybe it's useful.